app/components/billing-select-plan.js

```diff
@@ -0,0 +1,27 @@
+import Component from '@ember/component';
+import { inject as service } from '@ember/service';
+import { task } from 'ember-concurrency';
+
+export default Component.extend({
+
+  plan: service(),
+
+  showPlansSelector: true,
+  showCancelButton: false,
+  title: null,
+
+  save: task(function* () {
+    if (this.submit.perform) {
+      yield this.submit.perform();
+    } else {
+      this.submit();
```

Comment:
It should be just `this.submit()` which would execute assigned action. Also it would be useful to add a default blank method to avoid JS errors: `submit() {}`

Ember Concurrency exposes `{{preform <task>}} template helper which emits an action that calls `task.perform`. This helper should be used to bind tasks to child component actions.